docs/tutorial_analyzer.md

```diff
@@ -74,3 +74,121 @@ print(
                                               score=res.score)
       for res in results])
 ```
+
+## Using multiple languages
+
+### Option 1: Configure which spacy/stanza models to load
+
+First, we need to edit the configuration file (i.e. [/presidio-analyzer/conf/spacy_multilingual.yaml](../presidio-analyzer/presidio/conf/spacy_multilingual.yaml) ) to specify what languages to load. Like this:
+
+```spacy_multilingual.yaml
+nlp_engine_name: spacy
+models:
+  -
+    lang_code: en
+    model_name: en_core_web_lg
+  -
+    lang_code: es
+    model_name: es_core_news_md
+```
+
+In this configuration we use the `en_core_web_lg` spaCy model for English, and `es_core_news_md` spaCy model for Spanish. Presidio supports both [spaCy](https://spacy.io/usage/models) and [Stanza](https://github.com/stanfordnlp/stanza) models.
+
+NOTE: make sure that you have all your spacy models downloaded with a command like this:
+
+```bash
+python3 -m spacy download es_core_news_md
+```
+
+Then, we need to serve our models as showed before in Method 1.
+
+Now that our models is up and running, we can send PII text to it.
+
+_From another shell_
+
+You can make an English language query like before or use Spanish language query like this:
+
+```sh
+./presidio-analyzer analyze --text "Mi nombre es Francisco Pérez con DNI 55555555-K, vivo en Madrid y trabajo para la ONU." --fields "ES_NIF" "LOCATION" "PERSON" --language "es"
+````
+
+The expected result is:
+
+```json
+{
+  "analyzeResults": [
+    {
+      "field": {
+        "name": "ES_NIF"
+      },
+      "score": 1.0,
+      "location": {
+        "start": 37,
+        "end": 47,
+        "length": 10
+      }
+    },
+    {
+      "field": {
+        "name": "PERSON"
+      },
+      "score": 0.8500000238418579,
+      "location": {
+        "start": 13,
+        "end": 28,
+        "length": 15
+      }
+    },
+    {
+      "field": {
+        "name": "LOCATION"
+      },
+      "score": 0.8500000238418579,
+      "location": {
+        "start": 57,
+        "end": 63,
+        "length": 6
+      }
+    }
+  ],
+  "requestId": "d89ffd66-2adf-486f-a040-4e4f80af7a86"
+}
+```
+
+### Option 2: load models programmatically
+
+Use the analyzer Python code by importing `analyzer_engine.py`, `SpacyNlpEngine` and `RecognizerRegistry`
+
+```python
+from presidio_analyzer import AnalyzerEngine
+from presidio_analyzer.nlp_engine import SpacyNlpEngine
+from presidio_analyzer.recognizer_registry import RecognizerRegistry
+
+registry = RecognizerRegistry()
+nlp = SpacyNlpEngine({"en": "en_core_web_lg", "es": "es_core_news_md"})
+registry.load_predefined_recognizers(["es"], "spacy")
```

Comment:
The first query (line 172) will not work as you didn't load the "en" recognizers which include the PHONE_NUMBER
```suggestion
registry.load_predefined_recognizers(["en", "es"], "spacy")
```